docs/2-configuration.md

```diff
@@ -27,33 +27,35 @@ The format for specifying multiple values for options that allow it is:
+| `WG_PORT`                            | `--port`                            | `port`                         |          | `8000`                                       | The port the web ui will listen on (http)                                                                                                                                                                                                                          |
+| `WG_EXTERNAL_HOST`                   | `--external-host`                   | `externalHost`                 |          |                                              | The external domain for the server (e.g. www.mydomain.com)                                                                                                                                                                                                         |
+| `WG_STORAGE`                         | `--storage`                         | `storage`                      |          | `sqlite3:///data/db.sqlite3`                 | A storage backend connection string. See [storage docs](./3-storage.md)                                                                                                                                                                                            |
+| `WG_DISABLE_METADATA`                | `--disable-metadata`                | `disableMetadata`              |          | `false`                                      | Turn off collection of device metadata logging. Includes last handshake time and RX/TX bytes only.                                                                                                                                                                 |
+| `WG_ENABLE_INACTIVE_DEVICE_DELETION` | `--enable-inactive-device-deletion` | `enableInactiveDeviceDeletion` |          | `false`                                      | Enable/Disable the automatic deletion of inactive devices.                                                                                                                                                                                                         |
+| `WG_INACTIVE_DURATION`               | `--inactive-duration`               | `inactiveDuration`             |          | `4380h`                                      | The duration after which inactive devices are automatically deleted, if automatic deletion is enabled. A device is inactive if it has not been connected to the server for longer than the inactive duration. The duration format is the go duration string format |
```

Comment:
This also needs to be adapted to the new duration name -- `WG_INACTIVE_DEVICE_GRACE_PERIOD`.